libavformat/libavformat.a(allformats.o): In function `av_register_all':
libavformat/allformats.c:66: undefined reference to `_avs_demuxer'
libavformat/allformats.c:68: undefined reference to `_bfi_demuxer'
libavformat/allformats.c:86: undefined reference to `_filmstrip_muxer'
libavformat/allformats.c:87: undefined reference to `_flac_demuxer'

--- origsrc/gst-ffmpeg-0.10.11/gst-libs/ext/ffmpeg/libavformat/allformats.c	2010-07-15 14:31:59.000000000 -0500
+++ src/gst-ffmpeg-0.10.11/gst-libs/ext/ffmpeg/libavformat/allformats.c	2010-09-28 00:16:10.103485200 -0500
@@ -63,9 +63,7 @@ void av_register_all(void)
     REGISTER_MUXDEMUX (AVI, avi);
     REGISTER_DEMUXER  (AVISYNTH, avisynth);
     REGISTER_MUXER    (AVM2, avm2);
-    REGISTER_DEMUXER  (AVS, avs);
     REGISTER_DEMUXER  (BETHSOFTVID, bethsoftvid);
-    REGISTER_DEMUXER  (BFI, bfi);
     REGISTER_DEMUXER  (BINK, bink);
     REGISTER_DEMUXER  (C93, c93);
     REGISTER_DEMUXER  (CAF, caf);
@@ -83,8 +81,8 @@ void av_register_all(void)
     REGISTER_DEMUXER  (EA_CDATA, ea_cdata);
     REGISTER_MUXDEMUX (EAC3, eac3);
     REGISTER_MUXDEMUX (FFM, ffm);
-    REGISTER_MUXDEMUX (FILMSTRIP, filmstrip);
-    REGISTER_MUXDEMUX (FLAC, flac);
+    REGISTER_DEMUXER  (FILMSTRIP, filmstrip);
+    REGISTER_MUXER    (FLAC, flac);
     REGISTER_DEMUXER  (FLIC, flic);
     REGISTER_MUXDEMUX (FLV, flv);
     REGISTER_DEMUXER  (FOURXM, fourxm);
